src/modules/SX127x/SX127x.cpp

```diff
@@ -782,6 +782,24 @@ int16_t SX127x::setPreambleLength(size_t preambleLength) {
   return(RADIOLIB_ERR_UNKNOWN);
 }
 
+int16_t SX127x::setPreamblePolarity(int polarity) {
```

Comment:
Please add a call to `setPreamblePolarity` in `SX127x::beginFSK`, so that at reset, the value is always set to a known default.

Reply:
There is already a direct register SPIsetRegValue in configFSK which does the same. Could remove it here and put the new method in beginFSK.

Reply:
ok, done.

Reply:
That's exactly what I meant, thanks.